astropy/io/misc/asdf/tags/coords/angle.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from asdf.yamlutil import custom_tree_to_tagged_tree
+from astropy.coordinates import Angle, Latitude, Longitude
+
+from ..unit.quantity import QuantityType
+
+
+__all__ = ['AngleType', 'LatitudeType', 'LongitudeType']
+
+
+class AngleType(QuantityType):
+    name = "coords/angle"
+    types = ['astropy.coordinates.angles.Angle']
+    requires = ['astropy']
+    version = "1.0.0"
+    organization = 'astropy.org'
+    standard = 'astropy'
```

Comment:
Might be useful to have a comment here saying that we inherit from `QuantityType`, whose schema lives in `stsci.edu:asdf`, but we need to override that here since `AngleType`'s schema is provided by `astropy.org:astropy`.